sn_networking/src/record_store.rs

```diff
@@ -1201,6 +1205,147 @@ mod tests {
         assert!(store.get(&r.key).is_none());
     }
 
+    #[tokio::test]
+    async fn can_store_and_retrieve_chunk() {
```

Comment:
Is this test relevant to this PR?

Reply:
No, but was added as I added one for SP and thought it a good addition.